### PR TITLE
Feat/block local

### PR DESCRIPTION
**Pull request by cynthiaxie**
1. remove data upload
2. change block threshold millis to configurable
3. save block info to local files

### CODE REVIEW

File: Android/dokit/src/main/java/com/didichuxing/doraemonkit/datapick/DataPickManager.java [hunk 1]
```diff
@@ -117,33 +117,22 @@ public void postData() {
...
-        }, new Response.ErrorListener() {
-            @Override
-            public void onErrorResponse(VolleyError error) {
-                LogHelper.e(TAG, "error===>" + error.getMessage());
-                //ToastUtils.showShort("上传埋点失败");
-            }
-        });
-
-        VolleyManager.INSTANCE.add(requset);
+//        Request requset = new JsonObjectRequest(Request.Method.POST, NetworkManager.APP_DATA_PICK_URL, new JSONObject(content), response -> {
```
Comment (seazon) [inline Android/dokit/src/main/java/com/didichuxing/doraemonkit/datapick/DataPickManager.java:124]: remove these codes

**Review by seazon — CHANGES_REQUESTED** [review]

**Comment by seazon** [comment]
pls don't submit [Revert "feat: remove upload user info and data pick info and app heal…](https://github.com/Wiredcraft/DoKit/pull/2/commits/70c4320fe1253323dced9af79cbf8fdb7baa4de7) such revert code

your commits should be only 3
[feat: save block info to local files](https://github.com/Wiredcraft/DoKit/pull/2/commits/942c214e0ad5effb103269ce4da48c30fa831c16)
~[feat: remove upload user info and data pick info and app health info](https://github.com/Wiredcraft/DoKit/pull/2/commits/0e9447c08eca83388062c0067c5e7786cf4b29bc)~
[feat: change block threshold millis to configurable, config it when D…](https://github.com/Wiredcraft/DoKit/pull/2/commits/e2012d11c33c7661236218ec86fe6855f1d16fd7) 
~[Revert "feat: remove upload user info and data pick info and app heal…](https://github.com/Wiredcraft/DoKit/pull/2/commits/70c4320fe1253323dced9af79cbf8fdb7baa4de7)~
[feat: not upload data pick info、app base info and app health info any…](https://github.com/Wiredcraft/DoKit/pull/2/commits/46fdfab54fae75aa701a0e06915be46edcd2fabc) 

you can drop commit via Drop Commit in Android Studio Git view, or use Git command.

File: Android/dokit/src/main/java/com/didichuxing/doraemonkit/util/DoraemonStatisticsUtil.java [hunk 2]
```diff
@@ -50,24 +50,25 @@ public static void uploadUserInfo(Context context) throws Exception {
...
-            public void onFailure(Call call, IOException e) {
-                e.printStackTrace();
-            }
-
-            @Override
-            public void onResponse(Call call, Response response) throws IOException {
-                response.close();
-            }
-        });
+        //app 启动数据埋点
```
[the remark this answers is not in the document]
Comment (cynthiaxie) [reply Android/dokit/src/main/java/com/didichuxing/doraemonkit/util/DoraemonStatisticsUtil.java:53]: ok

**Comment by seazon** [comment]
new PR created: https://github.com/Wiredcraft/DoKit/pull/3

**Pull request closed without merging**